test/suites/ACID.m

```diff
@@ -2742,3 +2747,109 @@
   xlim([-1, 2]);
 end
 % =========================================================================
+function [stat] = colorbars()
+  stat.description = 'Manual positioning of colorbars';
+  stat.issues      = [933 937];
+
+  shift = [0.2 0.8 0.2 0.8];
+  axLoc = {'in','out','out','in'};
+
+  for iAx = 1:4
+    hAx(iAx) = subplot(2,2,iAx);
+    axPos    = get(hAx(iAx), 'Position');
+    cbPos    = [axPos(1)+shift(iAx)*axPos(3), axPos(2), 0.02, 0.2]; 
+
+    hCb(iAx) = colorbar('Position', cbPos, 'AxisLocation', axLoc{iAx});
+    title(sprintf('AxisLocation = %s', axLoc{iAx}));
+    grid('on');
+  end
+end
+% =========================================================================
+function [stat] = colorbarManualLocationRightOut()
+  stat.description = 'Manual positioning of colorbars';
+  stat.issues      = [933 937];
+
+  axLoc = 'out'; 
+
+  figure('Units','centimeters','Position',[1,1,11,10]);
```

Comment:
Please don't use `figure` in your tests: that is something that our test framework takes care of (as that allows us to properly close the figure after testing).

If you need to change the figure, you can always `set(gcf, ....)` to get the same effect.